src/dirigera/devices/light.py

```diff
@@ -30,21 +22,32 @@ class Light(Device):
     def refresh(self) -> None:
         data = self.dirigera_client.get(route=f"/devices/{self.device_id}")
         attributes: Dict[str, Any] = data["attributes"]
+
+        if "startupOnOff" in attributes:
+            startup_on_off = StartupEnum(attributes.get("startupOnOff"))
+        else:
+            startup_on_off = None
+
         self.device_id = data["id"]
         self.is_reachable = data["isReachable"]
         self.custom_name = attributes["customName"]
         self.is_on = attributes["isOn"]
-        self.startup_on_off = attributes.get("startupOnOff")
+        self.startup_on_off = startup_on_off
         self.light_level = attributes.get("lightLevel")
         self.color_temp = attributes.get("colorTemperature")
         self.color_temp_min = attributes.get("colorTemperatureMin")
         self.color_temp_max = attributes.get("colorTemperatureMax")
         self.color_hue = attributes.get("colorHue")
         self.color_saturation = attributes.get("colorSaturation")
         self.firmware_version = attributes.get("firmwareVersion")
+        self.hardware_version = attributes.get("hardwareVersion")
         self.room_id = data["room"]["id"]
         self.room_name = data["room"]["name"]
+        self.room_name = data["room"]["name"]
         self.can_receive = data["capabilities"]["canReceive"]
+        self.model=attributes.get("model")
```

Comment:
```suggestion
        self.model = attributes.get("model")
```
Run black once to make sure the style is correct